src/components/DefineDissolution/DestroyCertificate.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div>
+    <div v-if="isSummary" class="destroy-certificate-summary">
+      <div v-if="getHasCertificateDestroyed" class="d-flex">
+        <span>
+          <v-icon class="destroy-certificate-success-chk ml-n1 pr-2" color="successCheckmark">mdi-check</v-icon>
+        </span>
+        <span v-html="destroyCertificateDescription" />
+      </div>
+      <div v-else>(Not entered)</div>
+    </div>
+    <v-card v-else flat class="rounded-4">
+      <div class="section-container pt-11" :class="{ 'invalid-section': showErrorSummary }">
+        <v-row no-gutters>
+          <v-col md="3">
+            <label class="destroy-certificate-title">Delete and/or Destroy Certificates</label>
+          </v-col>
+          <v-col md="9">
+            <v-checkbox
+              class="destroy-certificate-checkbox mt-0"
+              v-model="hasCertificateDestroyed"
+              @change="setHasCertificateDestroyed"
+            >
+              <template slot="label">
+                <span class="destroy-certificate-option"
+                  v-html="destroyCertificateDescription"
+                  :class="{'error-text': showErrorSummary}" />
+              </template>
+            </v-checkbox>
+          </v-col>
+        </v-row>
+      </div>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Vue, Prop } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+
+// Interfaces
+import { ActionBindingIF } from '@/interfaces'
+
+@Component
+export default class DestroyCertificate extends Vue {
+  @Prop({ default: false })
+  private readonly showErrorSummary: boolean
+
+  @Prop({ default: false })
+  private readonly isSummary: boolean
+
+  private hasCertificateDestroyed: boolean = false
+
+  private destroyCertificateDescription: string = `Certificates of incorporation, name change, and amalgamation for
+    the Cooperative Association will be deleted and/or destroyed after dissolution.`
+
+  @Getter getHasCertificateDestroyed!: boolean
+
+  // Global setters
+  @Action setHasCertificateDestroyed!: ActionBindingIF
```

Comment:
Nit picking here, but usually define our vuex bindings above the local properties 😁